src/mono/browser/runtime/rollup.config.js

```diff
@@ -191,7 +196,24 @@ const runtimeConfig = {
         }
     ],
     external: externalDependencies,
-    plugins: [regexReplace(inlineAssert), regexCheck([checkAssert, checkNoLoader]), ...outputCodePlugins],
+    plugins: [regexReplace(inlineAssert), regexCheck([checkAssert, checkNoLoader, checkNoDiag]), ...outputCodePlugins],
+    onwarn: onwarn
+};
+const diagConfig = {
+    treeshake: !isDebug,
+    input: "diagnostics/index.ts",
+    output: [
+        {
+            format: "es",
+            file: nativeBinDir + "/dotnet.diag.js",
```

Comment:
nit: Do we want to go with shortcut `diag` instead of full `diagnostics`? With HG we also used `globalization` instead of eg `glob`